### DISCUSSION TITLE
Change base64 fallback image?

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#4355] [Q&A] [answered]
Hello! Is there a config to change these base64 fallback image?
I don't rebember if i have read something about it.
![image](https://user-images.githubusercontent.com/2287371/171055853-f0036c7e-b7c8-4c03-a727-eeab683f50d4.png)

**Comment by Ju99ernaut** [comment, marked as answer]
I think you'd need to extend the image component for that https://github.com/artf/grapesjs/blob/dev/src/dom_components/model/ComponentImage.js

**Comment by censonalvin** [comment]
hello i'm also getting this base64 on my images. is there a fix on this?

**Reply by ronaldohoch** [reply]
As @Ju99ernaut shows, you can extend the image component to change the src of it.

But, if you need a sample, i'm doing it here:

```js
    editor.BlockManager.add('image', {
      label: "...",
      category: "...",
      media: `...`,
      content: {
        type: 'image',
        activeOnRender: 1,
        src: editor.runCommand("dina:get-image-svg")
      }
    });
```

Where `editor.runCommand("dina:get-image-svg")` is a command that generates a custom svg with a translated text.

```js
   editor.Commands.add("dina:get-image-svg", (editor, sender, opts = {}) => {
        return 'the svg...';
   });
```
